expect/expect.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+//ISC License
```

Comment:
I think we can use some short form of license header like:

```
Copyright 2019 Allain Lalonde. All rights reserved. ISC License.
```

Reply:
Done !